src/OpenLoco/src/Graphics/Gfx.cpp

```diff
@@ -316,7 +318,35 @@ namespace OpenLoco::Gfx
     // 0x0046E07B
     void loadCurrency()
     {
-        call(0x0046E07B);
+        const auto* currencyObject = ObjectManager::get<CurrencyObject>();
+        if (currencyObject == nullptr)
+        {
+            return;
+        }
+
+        const auto loadCurrencySymbol = [](const auto font, const auto imageId, const auto dstImageId) {
+            const auto* image = getG1Element(imageId);
+            if (image == nullptr)
+            {
+                return;
+            }
+
+            auto* currencyElement = getG1Element(dstImageId);
+            if (currencyElement == nullptr)
+            {
+                return;
+            }
+
+            Gfx::setCharacterWidth(font, 131, image->width - 1);
```

Comment:
Do we not have a constant for index 131 yet? Perhaps we should add one.

Interestingly, in the Windows-1252 codepage, upon which the Locomotion text encoding seems to have been based, index 131 is used for the [Florin currency sign (ƒ)](https://en.wikipedia.org/wiki/Florin_sign).

Reply:
Not sure we have such a table, I think we do for the images but not the character set.

Reply:
Thats because its not 131 its 0xA3 which is the pound sign in Windows-1252.

Reply:
So can just be replaced by `u'£'`

Reply:
Why he didn't just use 0xA4 which is the general currency sign I don't know.

Reply:
You are right, given it had 131 in CurrencyObject I didn't really think about it that much, but it makes sense to use `u'£'`, I'll change it.